extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -76,10 +79,12 @@ export class AmpAnim extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    if (!this.srcset_) {
-      this.srcset_ = srcsetFromElement(this.element);
+    if (this.img_.src == SRC_PLACEHOLDER) {
```

Comment:
Nit: we can always propagate, it won't be that expensive.

Reply:
Done.